### PR TITLE
add XSeries and update Ponder

### PR DESCRIPTION
**Pull request by VoChiDanh**
ye maybe

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,13 +102,18 @@
...
+            <version>1.18.1-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>preponderous</groupId>
-            <artifactId>ponder</artifactId>
-            <version>v0.14-alpha-2</version>
+            <groupId>com.github.Preponderous-Software</groupId>
+            <artifactId>Ponder</artifactId>
+            <version>v1.0</version>
```
Comment (dmccoystephenson) [inline pom.xml:111]: This should be changed to 1.0. It should match the JAR.

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
Looks good! Just one minor thing to change.

File: pom.xml (hunk 1 above)
Comment (VoChiDanh) [reply pom.xml:111]: ah, i use jitpack so it v1.0

Comment (dmccoystephenson) [reply pom.xml:111]: Sounds good. Would you mind adding the jitpack information to the README?

**Review by dmccoystephenson — APPROVED** [review]

**Pull request merged**